daemon/cluster/executor/container/validate.go

```diff
@@ -1,43 +0,0 @@
-package container
-
-import (
-	"fmt"
-	"os"
-	"path/filepath"
-
-	"github.com/docker/swarmkit/api"
-)
-
-func validateMounts(mounts []api.Mount) error {
```

Comment:
I'm not sure I understand why the agent would do the validation when the underlying daemon is already going to do it.